sr_robot_commander/src/sr_robot_commander/sr_robot_commander.py

```diff
@@ -392,7 +392,14 @@ def make_named_trajectory(self, trajectory):
 
         time_from_start = 0.0
 
-        for wp in trajectory:
+        current_waypoint = {
+            'joint_angles': [],
+            'interpolate_time': 0.01,
+            'pause_time': 0.01
```

Comment:
Same here. We should make the pause_time=0.0 so that this waypoint doesn't introduce an unnecessary micro-stop of the robot.